sdk-core/src/main/java/care/data4life/sdk/attachment/AttachmentService.kt

```diff
@@ -36,11 +35,13 @@ class AttachmentService internal constructor(
     // TODO move imageResizer to thumbnail service
     private val imageResizer: AttachmentContract.ImageResizer
 ) : AttachmentContract.Service {
+    private val hasher: AttachmentContract.Hasher = AttachmentHasher
```

Comment:
I noticed this in several places, why not adding this to the constructor?

Reply:
Since, like in other modules, this could be also done by simply reference them, it's more kind of a marker. But you are right, these things should be go away in any place once we make use of Koin (hopefully very soon). However putting them in the constructor now is more of a test concern rather than a structural necessity.